### DISCUSSION TITLE
Deleting a plugin keymap doesn't work like described in guide

### DISCUSSION THREAD

**Discussion by pogopaule** [LazyVim/LazyVim#4567] [Q&A] [answered]
Following [the guide](https://www.lazyvim.org/configuration/plugins#%EF%B8%8F-adding--disabling-plugin-keymaps), I added in `plugins/overrides.lua`

```
  {
    "akinsho/bufferline.nvim",
    keys = {
      -- I want to use L and H to move the cursor to the top or bottom of the buffer
      { "<S-h>", false },
      { "<S-l>", false },
    },
  },
```

This does not work. Why?

I can't find anything in [bufferline's setup](https://github.com/LazyVim/LazyVim/blob/13a4a84e3485a36e64055365665a45dc82b6bf71/lua/lazyvim/plugins/ui.lua#L48-L49) that is a problem.

**Comment by dpetka2001** [comment, marked as answer]
That's because `<s-h>/<s-l>` are mapped in 2 places as you can see [here](https://github.com/search?q=repo%3ALazyVim%2FLazyVim%20%3Cs-h%3E&type=code). So, you also have to delete the mappings in your `keymaps.lua` file, like `vim.keymap.del("n", "<S-h>")`. When you want to disable keymaps just do a search in the repo to see where they are defined.

**Reply by pogopaule** [reply]
Thank you so much for your quick answer!
